Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -1,192 +1,77 @@
 /* Style applied to the TabStrip. */
 
-/* Applied to the tab strip itself.*/
-*.tabStrip
-{
-}
-
-*.tabStripTabsPane
-{
-}
-
-*.tabStripTabsPane.empty
-{
-}
-
-*.tabStripTabsPane ul
+.tabStripTabsPane ul
 {
+  padding: var(--remotion-themed-padding);
   margin: 0;
-  padding: 0px;
   list-style: none;
-  width: auto;
+  display: flex;
+  flex-flow: wrap;
+  gap: var(--remotion-themed-padding);
+  box-sizing: border-box;
 }
 
-*.tabStripTabsPane ul li
+.tabStripTabsPane ul li
 {
-  display: inline;
+  padding-top: calc(var(--remotion-themed-padding) / 2);
+  padding-bottom: calc(var(--remotion-themed-padding) / 2);
```

Comment:
Are you sure that those declarations are needed? When making the tabs wrap into a new line, the row gap is "much" larger than the column gap, due to this setting. Imo this can be deleted.